### DISCUSSION TITLE
Hide Work Space Tools when not in sketch mode

### DISCUSSION THREAD

**Discussion by SiboVG** [hlorus/CAD_Sketcher#357] [Feature Requests]
Hi, I just started playing around with Sketcher and I love it, it's really cool, thank you all for your hard work on this!

However, I do find it a bit strange that the Sketcher work space tools are visible all the time, even when you're not in sketch mode. It would make more sense to me to hide them and only show them in sketch mode. Going further, you could even hide some of the default Blender work tools, such as the object move tool etc., because they do not apply to objects in sketch mode. Don't know if that's even possible though, if not, just ignore me :).

**Comment by hlorus** [comment]
Unfortunately the API doesn't really allow this. Someone asked on DevTalk but still no answer:
https://devtalk.blender.org/t/possible-to-create-additional-viewport-modes-with-addon/24391?u=hlorus

What currently comes closest to this is the active addons setting per workspace which the user can find in the properties editor iirc.
